emmarin/cl/ledger/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ledger"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+cl = { path = "../cl" }
+ledger_proof_statements = { path = "../ledger_proof_statements" }
+nomos_cl_risc0_proofs = { path = "../risc0_proofs" }
+ledger_validity_proof = { path = "../ledger_validity_proof" }
+risc0-zkvm = { version = "1.0", features = ["prove", "metal"] }
```

Comment:
I'm not sure, but perhaps using the latest version of risc0 could impact efficiency. I think the current version is 1.1.3

Reply:
Thanks for the head up! FIY, `Cargo.toml` only selects the lowest compatible version. `Cargo.lock` contains the actual version used, and I was running tests with 1.1.2